python/py-curtsies/Portfile

```diff
@@ -15,23 +15,38 @@ description             Curses-like terminal wrapper, with colored strings
 long_description        ${description}
 
 homepage                https://curtsies.readthedocs.io
-master_sites            pypi:[string index ${python.rootname} 0]/${python.rootname}
-distname                ${python.rootname}-${version}
-checksums               rmd160  39fd8eb76f8e38fec3a7803659e3ae1a7cbfce2a \
-                        sha256  89c802ec051d01dec6fc983e9856a3706e4ea8265d2940b1f6d504a9e26ed3a9 \
-                        size    47120
 
-python.versions         27 35 36 37
+checksums               rmd160  b3107c9109491dccd1a16b5805c1063366574641 \
+                        sha256  a587ff3335667a32be7afed163f60a1c82c5d9c848d8297534a06fd29de20dbd \
+                        size    53472
+
+python.versions         27 36 37 38 39
 
 if {${name} ne ${subport}} {
     livecheck.type          none
 
     depends_build-append    port:py${python.version}-setuptools
 
-    depends_lib-append      port:py${python.version}-blessings \
+    depends_lib-append      port:py${python.version}-cwcwidth \
+                            port:py${python.version}-blessings
+
+    # Version >= 0.3.5 requires Python >= 3.6
+    if {${python.version} == 27} {
+        version             0.3.4
+        revision            0
+
+        patchfiles          patch-setup-io-open.diff
```

Comment:
The python 27 subport was working before (on 0.3.0 at least), and the patch is just changing the description. Would you mind explaining how this helps? Thanks.

Reply:
In the latest supported version for Python 2.7, the author used `open` instead of `io.open`, which does not work on Python 2.7. Everything else works as expected.